src/app/Plans/components/Wizard/VMConcernsDescription.tsx

```diff
@@ -0,0 +1,54 @@
+import { IVMwareVM } from '@app/queries/types';
+import { TextContent, Text, List, ListItem } from '@patternfly/react-core';
+import * as React from 'react';
+
+interface IVMConcernsDescriptionProps {
+  vm: IVMwareVM;
+}
+
+const criticalText = (
```

Comment:
This might a question of style but since `criticalText` and siblings are not just used directly since there are all used onces. I understand it might be for clarity but then I was uncomfortable using Fragment everywhere so I wonder if it makes more sense to transform it to a React Node element.

Reply:
Yeah, I only made them variables because the code was getting hard to read when they were inline. I'm not sure I know what you mean in terms of making them React Nodes, you mean components? The component version would still need to return a fragment, or something else like a span (the only reason I'm not using strings is so that I can render the `<strong>` in some of them)

Reply:
@gildub I added https://github.com/konveyor/virt-ui/pull/129/commits/f7f6eda428a6f5d7954e9423ef8611fb18446690 to clean this up a bit and consolidate the logic. Is that a bit better?

Reply:
@mturley, sorry about the confusion, you know between JSXElement,, ReactElement and ReactNode!

That is slicker!

Reply:
Heh yeah, I know. The TS types for React aren't perfect... I wish the React core team would maintain them, but they come from the DefinitelyTyped community :) Facebook still wants to try and push Flow for typing...

My understanding is, basically don't ever use JSX.Element or ReactElement, since they are specific to various kinds of components / React internals. I think the best practice is to pretty much always just use ReactNode when referring to a variable that contains JSX, since it's the most inclusive type among them (JSX.Element and ReactElement are both assignable to ReactNode), so it can store JSX no matter where it came from. Most of the PF components that work with values like this use ReactNode.

Reply:
I think even `null` is a valid `ReactNode` because you can return it from a component or render it as a child.